src/qt/forms/debugwindow.ui

```diff
@@ -1480,6 +1480,19 @@
                  </property>
                 </widget>
                </item>
+               <item row="18" column="0">
+                <spacer name="verticalSpacer_3">
+                 <property name="orientation">
+                  <enum>Qt::Vertical</enum>
+                 </property>
+                 <property name="sizeHint" stdset="0">
```

Comment:
Property could be removed.

Reply:
It seems indeed unneeded, I think it is how Qt creator creates spacers though. Change pushed.

Reply:
Hmm, I'm noticing other spacers have the property set, like the one in the information tab on the same file.